README.md

```diff
@@ -599,6 +599,7 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 * [XMind](http://www.xmind.net) - The most popular mind-mapping tool on the planet.
 * [Simple Diagrams](https://www.simplediagrams.com/) - A desktop app for creating hand-drawn-like, fast, clear sketches of problems, processes, workflows, ideas and more!
 * [yGraph Editor](https://www.yworks.com/products/yed) - High quality diagrams made easy.
+* [GitMind](https://gitmind.cn/) - The new generation of free mind map collaboration software.![Freeware][Freeware Icon]
```

Comment:
@cy920820  The website can only be accessed within China, so it can only be added to the Chinese list in README-zh.md.

Reply:
@jaywcjlove  It has been deleted, thank you